README.md

```diff
@@ -106,10 +106,10 @@ class App extends Component {
     this.state = {
       items: getItems(10)
     }
-    this.onDragEnd = this.onDragEnd.bind(this);
+    this.handleDragEnd = this.handleDragEnd.bind(this);
```

Comment:
This convention is not used in the code, examples or docs. For now we'll keep it consistent and just use on*

Also - I like using the on* rather than handle*! 😊